lib/codebreaker/hmi.rb

```diff
@@ -0,0 +1,89 @@
+module Codebreaker
+  class Hmi
+    def initialize
+      @game_stop = false
+      @hmi_commands = %w[help hint save replay exit show_score erase_score]
```

Comment:
It is cool that you know metaprogramming :) I accept your approach in this task. But in future try to not use metaprogramming. For example, instead of metaprogramming, you can use approach with describing all your commands in hash and then call this commands (useful article where you can read about this http://bobnadler.com/articles/2010/08/02/refactoring-case-statements-in-ruby.html).